mlrun/config.py

```diff
@@ -482,6 +482,10 @@
         "verbose": True,
         # the number of workers which will be used to trigger the start log collection
         "concurrent_start_logs_workers": 15,
+        # the time in hours in which to start log collection for runs when initialization API
+        # after upgrade we might have runs which completed in the mean time or still in non-terminal state and
+        # we want to collect their logs in the new log collection method (sidecar)
+        "log_buffer_start_collection_in_hours": 6,
```

Comment:
I understand the 2 last comment rows, but still not sure what this value is for.
Is the hours the time passed since the runs have completed? since the runs started?

Reply:
since the runs started